easybuild/easyconfigs/a/ALLPATHS-LG/ALLPATHS-LG-52488-foss-2015b.eb

```diff
@@ -0,0 +1,174 @@
+easyblock = 'ConfigureMake'
+
+name = 'ALLPATHS-LG'
+version = '52488'
+
+homepage = 'http://www.broadinstitute.org/software/allpaths-lg/blog/'
+description = """ALLPATHS-LG, the new short read genome assembler."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+# source is moved over time, hence multiple URLs below
+source_urls = [
+    'ftp://ftp.broadinstitute.org/pub/crd/ALLPATHS/Release-LG/latest_source_code',
+    'ftp://ftp.broadinstitute.org/pub/crd/ALLPATHS/Release-LG/latest_source_code/2015-04',
+    'ftp://ftp.broadinstitute.org/pub/crd/ALLPATHS/Release-LG/latest_source_code/2015/2015-04',
```

Comment:
If they make a new release and stick to their current logic, then the new release will first be in just plain vanilla latest and later on move to a YYYY-MM and later a YYYY/YYYY-MM sub dir. The version for which this EasyBuild recipe was made (52488) was released in April 2015 and will stay in 2015-04 (unless they re-organise the tree/layout of their server off course.)